src/Agent/NewRelic/Agent/Core/Agent.cs

```diff
@@ -322,10 +322,10 @@ public async Task TryInjectBrowserScriptAsync(string contentType, string request
             if (rumBytes == null)
             {
                 transaction.LogFinest("Skipping RUM Injection: No script was available.");
-                await baseStream.WriteAsync(buffer, 0, buffer.Length);
+                await baseStream.WriteAsync(buffer, 0, buffer.Length).ConfigureAwait(false);
```

Comment:
I don't know if this change is safe or not, because it may rely on data from the synchronization context.